### PR TITLE
regexp-in text: add tests for empty pattern with sequential search

### PR DESCRIPTION
**Pull request by komainu8**

### CODE REVIEW

File: sql/regexp/text/regexp-in/empty-element/only/seqscan.sql [hunk 1]
```diff
@@ -0,0 +1,22 @@
+CREATE TABLE memos (
+  content text
+);
+
+INSERT INTO memos VALUES ('PostgreSQL is a RDBMS');
+INSERT INTO memos VALUES ('Groonga is fast full text search engine');
+INSERT INTO memos VALUES ('PGroonga is a PostgreSQL extension that uses Groonga');
```
Comment (otegami) [inline sql/regexp/text/regexp-in/empty-element/only/seqscan.sql:7]: How about removing these two records because I think one record is enough for this test pattern?

File: sql/regexp/text/regexp-in/empty-element/mixed/one-pattern/seqscan.sql [hunk 2]
```diff
@@ -0,0 +1,22 @@
+CREATE TABLE memos (
+  content text
+);
+
+INSERT INTO memos VALUES ('PostgreSQL is a RDBMS');
+INSERT INTO memos VALUES ('Groonga is fast full text search engine');
+INSERT INTO memos VALUES ('PGroonga is a PostgreSQL extension that uses Groonga');
```
Comment (otegami) [inline sql/regexp/text/regexp-in/empty-element/mixed/one-pattern/seqscan.sql:7]: How about removing this record?

**Review by otegami — APPROVED** [review]
LGTM 👍🏾

File: sql/regexp/text/regexp-in/empty-element/mixed/two-patterns/seqscan.sql [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+SET enable_bitmapscan = off;
+
+EXPLAIN (COSTS OFF)
+SELECT *
+  FROM memos
+ WHERE content &~| ARRAY['', 'rdbms\z', '\Agroonga'];
+
+SELECT *
+  FROM memos
+ WHERE content &~| ARRAY['', 'rdbms\z', '\Agroonga'];
```
Comment (abetomo) [inline sql/regexp/text/regexp-in/empty-element/mixed/two-patterns/seqscan.sql:20]: いつもemptyが先頭なんですが、順番は気にしなくて良いですか？

Comment (komainu8) [reply sql/regexp/text/regexp-in/empty-element/mixed/two-patterns/seqscan.sql:20]: 順番によって処理が変わるわけではないので、先頭のemptyが無視されるなら他のパターンでも同様と判断して、先頭のみのテストにしています。

File: expected/regexp/text/regexp-in/empty-element/mixed/two-patterns/seqscan.out [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+ WHERE content &~| ARRAY['', 'rdbms\z', '\Agroonga'];
+                           QUERY PLAN                           
+----------------------------------------------------------------
+ Seq Scan on memos
+   Filter: (content &~| '{"","rdbms\\z","\\Agroonga"}'::text[])
+(2 rows)
+
+SELECT *
+  FROM memos
+ WHERE content &~| ARRAY['', 'rdbms\z', '\Agroonga'];
```
Comment (kou) [inline expected/regexp/text/regexp-in/empty-element/mixed/two-patterns/seqscan.out:22]: 最後じゃないところにRDBMSが入っているテストデータを追加できない？

Comment (komainu8) [reply expected/regexp/text/regexp-in/empty-element/mixed/two-patterns/seqscan.out:23]: 追加しました。
